frontend/javascripts/dashboard/dataset/queries.tsx

```diff
@@ -496,11 +496,9 @@ function diffDatasets(
     .map((id) => newDatasetsDict[id])
     .filter((newDataset) => {
       const oldDataset = oldDatasetsDict[newDataset.id];
-      return !_.isEqualWith(oldDataset, newDataset, (oldValue, newValue, key) => {
-        const didUpgradeToRenamableDS =
-          !("directoryName" in oldValue) && "directoryName" in newValue; // TODO: Can be remove after a few weeks / months.
-        if (key === "lastUsedByUser" || didUpgradeToRenamableDS) {
```

Comment:
what will happen if a user hasn't logged into wk for the past few months and logs in now?

Reply:
I'd say that all dataset are maked as diffs and thus are listed in the notification toast. 
See: https://github.com/scalableminds/webknossos/blob/5447c6e3f474f057a9ab55b9ef3ae4b77e81d88c/frontend/javascripts/dashboard/dataset/queries.tsx#L518-L549
And 
https://github.com/scalableminds/webknossos/blob/5447c6e3f474f057a9ab55b9ef3ae4b77e81d88c/frontend/javascripts/dashboard/dataset/queries.tsx#L204-L215

So it might be confusing :shrug:. Alternatively we need to maintain this check for ever or at least some more months :thinking:

Reply:
ok, I think, this is okay as it's low impact :)